test/integration/emailjs-imap-client-test.js

```diff
@@ -327,27 +333,27 @@
                     }).then(() => {
                         return imap.selectMailbox('inbox');
                     }).then((resultInfo) => {
-                        expect(initialInfo.exists !== resultInfo.exists).to.be.true;
-                    }).then(done).catch(done);
+                        expect(initialInfo.exists - 1 === resultInfo.exists).to.be.true;
```

Comment:
Why? Initial number minus one deleted gives final number. It passes, at least ;)